BandNameGenerator/Formula.cs

```diff
@@ -13,15 +13,22 @@ public enum Part
             Adjective,
             SingularNounPossessive,
             Number,
-            Verb,
-            InitialThe,
-            OfThe
+            ThirdPersonSingularVerb,
+            ThirdPersonPluralVerb,
+            TransitiveVerb,
+            Frequency,
+            The,
+            OfThe,
+            MeA,
+            Dont
         }
 
         private readonly List<Part> parts;
 
         private readonly static List<string> numbers = new List<string>() { "3", "12", "100", "500", "A Thousand" };
 
+        private readonly static List<string> frequencies = new List<string>() { "Always", "Never", "Seldom" };
+
         public Formula(List<Part> parts)
```

Comment:
Unnecessary idea from me: You could replace `List<Part>`, here with an array and add the [params](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/params) key word. Then it's as if the method takes an unlimited number of arguments, each of type `Part`. It avoids the caller having to do `new List<Formula.Part>(){...}` in cases like this:

```c#
numberPluralNoun = new Formula(new List<Formula.Part>(){ Formula.Part.Number, Formula.Part.PluralNoun });
```
Which could then be:
```c#
numberPluralNoun = new Formula(Formula.Part.Number, Formula.Part.PluralNoun);
```

Reply:
I like it!